### DISCUSSION TITLE
[Bug]: Unable to open uploaded video on first time

### DISCUSSION THREAD

**Discussion by vignesh-happiestminds** [pimcore/pimcore#13138] [Q&A / Support]
### Expected behavior

I have recently updated pimcore version 10.5.5. 

When i upload the video into assets. On click of the video, video should play. 

![Screenshot from 2022-09-13 11-15-43](https://user-images.githubusercontent.com/109342064/189819162-c57c7f9a-1992-4353-9008-c677da360f3d.png)

### Actual behavior

I have recently updated pimcore version 10.5.5. After update the latest version pimcore, facing permission issues.

When i upload the video into assets. On click of the video, video should play. Instead of that it is throwing an issues.

"An exception has been thrown during the rendering of a template ("Unable to create a directory at /var/www/html/project/public/var/tmp/asset-cache/1387/image-thumb__1387__video_original_image. mkdir(): Permission denied").
"
![Screenshot from 2022-09-13 11-08-41](https://user-images.githubusercontent.com/109342064/189819723-8bc07ac2-196f-4bd0-a739-ab512999ff04.png)

**### After closing the video tab and  reopening the video, it is working .**

![Screenshot from 2022-09-13 11-15-43](https://user-images.githubusercontent.com/109342064/189819162-c57c7f9a-1992-4353-9008-c677da360f3d.png)

### Steps to reproduce

- Login into application as admin
- Click on Assets Menu
- Under Asset - Click on **Add Asset** -> **Upload File** - Select video file
- Open the uploaded video

**Comment by dvesh3** [comment]
please provide file system permissions like describe here https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html

**Reply by vignesh-happiestminds** [reply]
I have executed below command

**sudo chown -R www-data:www-data var public/var**

But whenever we upload the video, it is created folder with user permission not with apache permission. Also this issue coming only after updated **pimcore version into 10.5.5**

Please find the below reference folder id **1396** which got generated while uploading video into asset.

![Screenshot from 2022-09-13 12-45-07](https://user-images.githubusercontent.com/109342064/189835460-27627a9b-8772-4350-a701-ff17adda7234.png)

**Comment by vignesh-happiestminds** [comment]
We have updated permission _**chown -R www-data:www-data var public/var**_.  Also we have checked on pimcore 10.5.7. Still this issue exists.
